src/data/logosCompanies.json

```diff
@@ -124,5 +124,12 @@
     "link": "https://app.danogo.io/about-us",
     "knownFor": " Danogo",
     "showCompanyName": false
+  },
+  {
+    "companyName": "No Witness Labs",
+    "imageName": "nowitnesslabs",
+    "link": "https://nowitnesslabs.com/",
+    "knownFor": "No Witness Labs",
```

Comment:
No Witness Labs cannot be known for No Witness Labs, it should rather name one of the tools/products or services on https://developers.cardano.org/showcase or https://developers.cardano.org/tools which is one of the requirements listed on https://cardano.org/docs/tutorial/add-company#requirements.

In this case I assume best is "Evolution SDK"? Thanks